pkg/kubectl/scale.go

```diff
@@ -165,8 +165,12 @@ func (scaler *ReplicationControllerScaler) ScaleSimple(namespace, name string, p
 			return err
 		}
 	}
-	controller.Spec.Replicas = int32(newSize)
-	if _, err := scaler.c.ReplicationControllers(namespace).Update(controller); err != nil {
+	scale := extensions.Scale{
+		ObjectMeta: api.ObjectMeta{Name: name, Namespace: namespace},
+		Spec:       extensions.ScaleSpec{Replicas: int32(newSize)},
+	}
+	if _, err := scaler.c.Extensions().Scales(namespace).Update("ReplicationController", &scale); err != nil {
```

Comment:
Should I do the same for the others?
I am thinking that I should also change the ScalePrecondition to accept a Scale rather than the various types. It would cut out an extra GET which isn't very useful.

WDYT?